### ISSUE TITLE
Access To Controls

### ISSUE THREAD

**Issue by KD33** [Excel-DNA/ExcelDna#303]
Hello,

I am trying to create a ComboBox for my addin.  I came across this:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.office.tools.excel.controls.combobox?view=vsto-2017

Which requires me to have access to my worksheet's controls.  Is this possible with Excel DNA?
From my understanding, controls are accessible as from Globals, which Excel DNA does not expose.

Is it possible for me to get access  to Controls in Excel DNA?  Or is there another way I can make the ComboBox?

Basically, I need a dropdown that will be pre-populated with some values.  Data Validation won't work because the dropdown characteristic doesn't show unless the cell is selected.  I'm also not sure how to handle the changing of values in a validation.

Thanks

**Comment by rkapl123** [comment]
Well, one (for ActiveX comboboxes) way is to access the object via `ActiveSheet.Shapes.Range("NameOfTheComboBox")(1).OLEFormat.Object`
Translated to ExcelDNA (VB.NET) this should be `ExcelDnaUtil.Application.ActiveSheet.Shapes.Range("NameOfTheComboBox")(1).OLEFormat.Object`

For a more elaborate usage of existing controls see [the assignHandler method of my DB-Addin project here](https://github.com/rkapl123/DBAddin/blob/master/source/AddInEvents.vb)
